### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by reubenjnazareth**

### CODE REVIEW

File: components/alert.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.alert-color1 {
+    background-color: var(--secondary-color);
+}
+
+.alert-blue {
+    border-color: rgb(25, 152, 202);
+    background-color: skyblue;
+
```
Comment (yashpatel1797) [inline components/alert.css:17]: avoid different method to assign value. try to stick with any one method.

File: alert.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+<div class="sub-title">Simple Alert</div>
+<div class="alert-nav">
+    <div class="alert-simple alert-color1">This is a simple alert.</div>
+    
+
+</div>
+
+<iframe
+  src="https://carbon.now.sh/embed?bg=rgba%2843%2C45%2C66%2C1%29&t=seti&wt=none&l=auto&width=800&ds=true&dsyoff=12px&dsblur=84px&wc=true&wa=false&pv=25px&ph=25px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522alert-simple%2522%253EThis%2520is%2520a%2520simple%2520alert.%253C%252Fdiv%253E"
```
Comment (yashpatel1797) [inline alert.html:55]: before pushing code you can do word wrap.

**Review by yashpatel1797 — APPROVED** [review]
added few suggestions look into it.

File: alert.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    <div class="alert-icon-warning alert-yellow alert-simple">Warning Alert</div>
+</div> -->
+
+
+
+
+
+
+    
+
```
Comment (adith-a-danthi) [inline alert.html:91]: You can remove commented code and blank lines

File: components/alert.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
+.alert-simple {
+   padding: 0.5rem;
+   border-style: solid;
+   border-width: 0.15rem;
```
Comment (adith-a-danthi) [inline components/alert.css:4]: You can use shorthand property. For small values say 4px or smaller you can use px instead of rem

**Review by adith-a-danthi — COMMENTED** [review]
Looks Good. Would suggest formatting code using prettier or other similar tools.

**Comment by reubenjnazareth** [comment]
yet to make corrections

**Pull request merged**